src/Assets/Javascript/scroll-tracking.js

```diff
@@ -0,0 +1,40 @@
+function ScrollTracking($module) {
+  this.$module = $module;
+}
+
+const triggerAnalyticsEvent = (category, action) => {
+  ga("send", "event", {
+    eventCategory: category,
+    eventAction: action,
+    transport: "beacon",
+    nonInteraction: true
+  });
+};
+
+ScrollTracking.prototype.init = function() {
+  var $module = this.$module;
+
+  if (!$module) {
+    return;
+  }
+
+  document.addEventListener("scroll", () => {
```

Comment:
I think we want to `throttle` this handler, because it runs operations like `document.body.scrollTop` which [force a layout and can cause performance issues](https://gist.github.com/paulirish/5d52fb081b3570c81e3a).